app/views/candidate_interface/application_form/success.html.erb

```diff
@@ -0,0 +1,11 @@
+<% content_for :title, page_title('Submit application') %>
+
+<div class="govuk-panel govuk-panel--confirmation govuk-!-margin-bottom-8">
+  <h1 class="govuk-panel__title">
+    Application successfully submitted
+  </h1>
+
+  <div class="govuk-panel__body">
+    Your application number is 61EQK
```

Comment:
We don't really have an application number in the system. @paulrobertlloyd what did we envision this being used for?

Reply:
that was left there for the show&tell,  I'll remove it for now, we'll think about it.

Reply:
We do need an application number, so that candidates can use this when contacting support). Ideally, it’s format will be different to that used by UCAS so that it’s easier to distinguish which system somebody’s application sits in. Does this need to be captured somewhere?

Reply:
I'll add a new card for this.